src/QsCompiler/Transformations/Monomorphization.cs

```diff
@@ -15,10 +15,9 @@
 
 namespace Microsoft.Quantum.QsCompiler.Transformations.Monomorphization
```

Comment:
Can you please add a documentation comment on the `Apply` public static method?

Reply:
Sure, most of the explanation is on the class doc, so the doc for the Apply method will be minimal.